packages/security/src/deviceTrust/SecurityCheckResult.ts

```diff
@@ -1,4 +1,19 @@
+/**
+ * Interface for the results of a single pass/fail security check.
+ */
 export interface SecurityCheckResult {
+  /**
+   * The name of the check. This can be used for reporting the check results.
+   */
   name: string;
+
+  /**
+   * Whether the check passed or failed. A successful check means that the environment this
```

Comment:
```
Whether the check passed or failed. A successful check means that the environment this application is running in is more secure than otherwise, as opposed to signalling if a certain feature was enabled
```
How about:

Flag that determines if executed check finished with success or failure.